xbmc/android/jni/Intent.h

```diff
@@ -35,6 +35,7 @@ class CJNIIntent : public CJNIBase
   std::string getType() const ;
 
   int getIntExtra(const std::string &name, int defaultValue) const;
+  std::string getStringExtra(const std::string &name) const;
```

Comment:
I thought that I'd need it, but finally not.
When this happen in the jni library, I tend to leave it there because the end goal is to have it as complete as possible, so any enrichment is a step in the right direction, even if not actually currently used, and removing it is wasted work that might have to be redone in the future